portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -108,6 +112,19 @@ private int getNumberOfComments(String numberOfCommentsString) {
     return numComments;
   }
 
+  /*
+   * Gets the direction comments should be sorted by based on string parameter.
+   * Defaults to DESCENDING (newest first) if argument is null or empty.
+   * @return sort direction
+   */
+  private SortDirection getSortDirection(String sortDirectionString) {
+    if (sortDirectionString == null || sortDirectionString.isEmpty() || sortDirectionString.equals("newest")) {
+      return SortDirection.DESCENDING;
+    } else {
```

Comment:
You don't need an else block if each conditional statement returns a value and ends the function early.

Reply:
Right good point